basket/BABY_LOUIE/src/js/app.js

```diff
@@ -1,16 +1,16 @@
 import $ from 'jquery';
-import whatInput from 'what-input';
+// import whatInput from 'what-input';
```

Comment:
Can this comment be removed?

Reply:
I don't know if this is needed as part of other remnants. So I left it commented. I'll remove it since it's easy enough to add back if needed.